spec/oncocore_stage.txt

```diff
@@ -7,7 +7,7 @@ CodeSystem:        LNC = http://loinc.org
                    //---------- General definitions of Stage Group and Stage Category ------
 Entry:             CancerStageGroup
 Parent:            CodedNonLaboratoryObservation
-Description:       "A panel that contains cancer staging information. This class is the parent of panels containing information related to clinical and pathologic staging, including TNMClinicalStageGroup and TNMPathologicStageGroup."
+Description:       "A panel that contains cancer staging information. This class is the parent of classes containing information related to clinical and pathologic staging, including TNMClinicalStageGroup and TNMPathologicStageGroup."
```

Comment:
More of an FYI: I thought you wanted to avoid mentioning "classes" in the context of FHIR. We'll keep this though.